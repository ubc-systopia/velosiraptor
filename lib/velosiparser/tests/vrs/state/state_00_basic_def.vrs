segment MyUnit {
  state = StateDef(base: foo) {
    reg foo [ 8 ],
    mem bar [ base, 0, 8 ],
  };
}
