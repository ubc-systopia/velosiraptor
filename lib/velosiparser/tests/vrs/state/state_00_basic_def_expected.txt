VelosiParseTree(tests/vrs/state/state_00_basic_def.vrs)
---------------------------------------------
segment MyUnit {
  state = StateDef(base: foo) {
    reg foo [ 8 ],
    mem bar [ base, 0, 8 ],
  };
}


---------------------------------------------
